app/src/main/java/org/mozilla/fenix/ext/Context.kt

```diff
@@ -100,3 +104,20 @@ fun Context.getRootView(): View? =
 
 fun Context.settings(isCrashReportEnabledInBuild: Boolean = BuildConfig.CRASH_REPORTING && Config.channel.isReleased) =
     Settings.getInstance(this, isCrashReportEnabledInBuild)
+
+fun Context.getStringWithArgSafe(@StringRes resId: Int, formatArg: String?): String {
```

Comment:
small nit: Would be great to add some tests for this.
I see though that other methods in this class are not tested so it's up to you.